interfaces/builtin/opengl.go

```diff
@@ -227,6 +227,9 @@ var openglConnectedPlugUDev = []string{
 
 	// Nvidia dma barrier
 	`SUBSYSTEM=="host1x-fence"`,
+
+	// Kernel Fusion Driver
+	`SUBSYSTEM=="kfd", GROUP="render", MODE="0660"`,
```

Comment:
this should be enough:
```suggestion
	`SUBSYSTEM=="kfd", KERNEL=="kfd"`,
```

the `GROUP` and `MODE` should not be used. We are not trying to override any policies but rather only match a given device so that an appropriate tag can be added. 

If you rebuild snapd locally and run it with the changes from this PR, you should see a snap specific tag, eg:
```
maciek@galeon:~ udevadm info /dev/dri/renderD12*
P: /devices/pci0000:00/0000:00:01.1/0000:01:00.0/0000:02:00.0/0000:03:00.0/drm/renderD128
M: renderD128
R: 128
U: drm
T: drm_minor
D: c 226:128
N: dri/renderD128
L: 0
S: dri/by-path/pci-0000:03:00.0-render
E: DEVPATH=/devices/pci0000:00/0000:00:01.1/0000:01:00.0/0000:02:00.0/0000:03:00.0/drm/renderD128
E: DEVNAME=/dev/dri/renderD128
E: DEVTYPE=drm_minor
...
E: CURRENT_TAGS=:snap_graphics-debug-tools-bboozzoo_vulkaninfo:...
```